tests/test_features_generator.py

```diff
@@ -47,7 +47,7 @@ def test_generate_features_with_default_parameters():
     feature_defs = gen_features(columns=columns, classes=[MockClass])
     assert len(feature_defs) == len(columns)
 
-    feature_dict = dict(feature_defs)
+    feature_dict = dict([_[0:2] for _ in feature_defs])
```

Comment:
We probably want to add an assertion checking that the default third argument is an empty dict.

Reply:
yes